src/Commands/BuildCommand.php

```diff
@@ -33,6 +33,12 @@ class BuildCommand extends Command
      */
     private $disk;
 
+    /**
+     * User ibis.php config settings
+     * @var array
+     */
+    private $_config;
```

Comment:
This `_` is weird. The times when underscores were used to mark variables as private are long gone. We should go with simple `$config` if we follow that approach.

Reply:
Cleaned this one up in latest push.